src/test/java/com/indeed/operators/rabbitmq/reconciliation/TestRabbitMQClusterReconciler.java

```diff
@@ -56,12 +65,12 @@
     private RabbitMQClusterReconciler reconciler;
 
     @BeforeEach
-    public void setup() {
```

Comment:
I'm a big believer in minimum access and technically these don't need to be public.

Reply:
yes very good